src/photos.cs

```diff
@@ -1108,10 +1150,22 @@ interface PHLivePhotoEditingContext {
 		[Export ("prepareLivePhotoForPlaybackWithTargetSize:options:completionHandler:")]
 		void PrepareLivePhotoForPlayback (CGSize targetSize, [NullAllowed] NSDictionary<NSString, NSObject> options, Action<PHLivePhoto, NSError> handler);
 
+#if false
```

Comment:
@dalexsoto are we dealt with this case yet ? i.e. creating a strong dict based on the generic nsdictionary